rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,71 @@
+# RFC: Communicating new package structure for `@fluentui/*`
+
+**Note that this RFC focuses solely on the package structure from a consumer perspective, focused primarily on v7/8 and converged packages. It does not deal with folder structure.**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+<!-- Explain the proposed change -->
+
+## Background
+
+(See also https://github.com/microsoft/fluentui/wiki/Repo-structure)
+
+**@fluentui/react** exports official finished components which we plan to support long-term (or provide good migration paths if we make major changes). It's mostly a re-export package, but a few components live here to avoid circular dependencies
+
+**@fluentui/react-internal** was split out solely to work around circular dependency issues, and most of the code for old versions of components lives here. Nobody should be referencing it directly outside our repo. `@fluentui/react` re-exports everything from this package (at least today), and some of the `@fluentui/react-<component>` packages depend on this package.
+
+**@fluentui/react-button** was previously considered ready for release (and therefore ready to be exported from @fluentui/react), but now we've decided that while it's ready for partners to try out and should be stable within this major release, there will probably be some significant changes in the next major release, so we're choosing not to re-export these components in the suite package anymore.
+
+**@fluentui/react-cards** is a weird case since there's an old version (built on @uifabric/foundation createComponent patterns) but it never officially graduated from experimental status and isn't exported from the suite. It also has a "next" folder where the converged version is currently being worked on.
```

Comment:
```suggestion
**@fluentui/react-cards** is a weird case since there's an old version (built on `@uifabric/foundation` createComponent patterns) but it never officially graduated from experimental status and isn't exported from the suite. It also has a "next" folder where the converged version is currently being worked on.
```